ui/src/semantic-ui/site/globals/site.overrides

```diff
@@ -152,4 +152,76 @@ form {
 
   .generate-margins(20, right, r);
   .generate-margins(20, left, l);
+
+  .bo-details {
```

Comment:
this is very specific to details pages, I think it should be used with some kind of component, f.e `Container` and then overwritten in the container.overrides
But I can fix it since I will continue with the details pages